### PR TITLE
EZP-27948: Add information regarding swapping Locations with the same name and parent

### PR DESCRIPTION
**Pull request by natanael89**
JIRA issue: [EZP-27948](https://jira.ez.no/browse/EZP-27948)
Related PR: https://github.com/ezsystems/ezpublish-kernel/pull/2297

This PR adds a description of the expected results when swapping Locations, especially with the same name and parent.
I am adding this to the User Documentation instead of Developer Documentation because only the former mentions swapping Locations.

### CODE REVIEW

File: docs/organizing_the_content.md [hunk 1]
```diff
@@ -41,6 +41,12 @@ and combining them with [proper permissions](organizing_the_site.md#permissions)
 You can swap two Content items in their Locations by going to the **Location Content Swap** section
 in the Location tab and selecting a Content item to swap with.
 
+!!! caution
+
+    Swapping Locations will also republish their respective URL aliases. This means that if the swapped Content Object was accessible by the URL that had a number at the end (added due to a name conflict), then after the swap the number will be removed (if possible). The opposite case also applies.
+    Swapping Locations for Content Objects with the same name and parent, results in swapping their URL aliases. This means that if the Content Object was previously accessible by a URL without a number at the end, after the swap it will be accessible by a URL with a number at the end (and the other way around).
```
Comment (DominikaK) [inline docs/organizing_the_content.md]: `Content Object` -> `Content item`, also in the line below

**Comment by bdunogier** [comment]
I'm thinking that an example of such a scenario (conflict, the end of the doc) would help. Do you agree ?

**Comment by natanael89** [comment]
@bdunogier Do you mean that we should provide an example how eZ behaves when there is a name conflict? I guess it would be a good idea to describe that in a place where we describe how Content name is used (and how it relates to the generated URL), but I have a hard time finding such place in our documentation. @MagdalenaZuba can you help?

**Comment by natanael89** [comment]
@bdunogier If you meant an example of the scenario described at the end of what is added in this PR, then I think this is such rare case that I wouldn't clog the user documentation more than it is absolutely necessary. @MagdalenaZuba @DominikaK what do you think?

**Comment by MagdalenaZuba** [comment]
I agree, if it is not a common problem there is no need to create artificial example just for documentation purposes.

**Comment by bdunogier** [comment]
> I think this is such rare case that I wouldn't clog the user documentation more than it is absolutely necessary. @MagdalenaZuba @DominikaK what do you think?

Then is it worth having a large paragraph in the middle of that doc for a rare edge case ? I was suggesting an example to make this complex paragraph easier to understand. Maybe the issue is actually at a higher level, and we should reconsider going that deep here.

An option is also to link to JIRA issues, as long as they clearly describe the bug and its solution.

**Comment by natanael89** [comment]
Unfortunately, link to JIRA issue is not an option, since the issue describes a more convoluted bug, and we want to describe just the end result (and the special case in the last paragraph).
We should probably find a better place for this description, but at the moment no such place exists - swapping Locations is not really document anywhere else. Therefore I would suggest leaving it here for now and eventually move the block to another place (after it is created).
As for adding an example, I have no strong feelings either way. Apparently, it would make the whole block easier to understand, but as I said I would leave it as short as possible in the current place. Probably it would be a good idea to expand on it after the dedicated place for swapping Locations is created.
I'm leaving this for the doc team to decide.

**Pull request merged**